pkg/apis/ceph.rook.io/v1/pool.go

```diff
@@ -46,6 +46,10 @@ func (p *ReplicatedSpec) IsTargetRatioEnabled() bool {
 	return p.TargetSizeRatio != 0
 }
 
+func (p *PoolSpec) IsMirroringEnabled() bool {
```

Comment:
i'd suggest we don't need wrapper methods like this, if it is a simple property being accessed.